javascript/i18n/phonenumbers/asyoutypeformatter_test.js

```diff
@@ -1150,37 +1150,44 @@ function testAYTFShortNumberFormattingFix_US() {
   assertEquals('1 22', f.inputDigit('2'));
 }
 
-function testAYTFClearNDDAfterIddExtraction() {
-  /** @type {i18n.phonenumbers.AsYouTypeFormatter} */
-  var f = new i18n.phonenumbers.AsYouTypeFormatter(RegionCode.KR);
-
-  // Check that when we have successfully extracted an IDD, the previously
-  // extracted NDD is cleared since it is no longer valid.
-  assertEquals('0', f.inputDigit('0'));
-  assertEquals('00', f.inputDigit('0'));
-  assertEquals('007', f.inputDigit('7'));
-  assertEquals('0070', f.inputDigit('0'));
-  assertEquals('00700', f.inputDigit('0'));
-  assertEquals('0', f.getExtractedNationalPrefix_());
```

Comment:
Or better, just remove these two lines. I don't see how this can work with closure strict compilation mode.